daisy_workflows/image_build/windows/post_install.ps1

```diff
@@ -359,6 +359,43 @@ function Set-Repos {
   Run-Command 'C:\ProgramData\GooGet\googet.exe' -root 'C:\ProgramData\GooGet' addrepo 'google-compute-engine-stable' 'https://packages.cloud.google.com/yuck/repos/google-compute-engine-stable'
 }
 
+function Export-ImageMetadata {
+  $version = (Get-ComputerInfo).OsVersion
+  $family = 'windows-' + (Get-ComputerInfo).windowsversion
+  $name =  (Get-ComputerInfo).OSName
```

Comment:
Any ideas on testing this? (I don't know powershell very well, so I'm not sure how feasible that'd be)

Reply:
haven't thought about testing. I can check if add this test somewhere in integration test, but let's leave as an extra change.